l10n_br_nfse/__manifest__.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 KMEE INFORMATICA LTDA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Edoc Nfse',
+    'summary': """
+        NFS-E""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'KMEE, Odoo Community Association (OCA)',
+    'maintainers': ['gabrielcardoso21', 'mileo', 'luismalta'],
+    'website': 'https://github.com/OCA/l10n-brazil',
+    'external_dependencies': {
+        'python': [
+            'erpbrasil.edoc',
+            'erpbrasil.assinatura',
+            'erpbrasil.transmissao',
+            'erpbrasil.base',
+        ],
+    },
+    'depends': [
+        'l10n_br_fiscal',
+        'l10n_br_zip',
```

Comment:
Existe a necessidade desta dependencia?

Reply:
Se o CEP for carregador outro módulo como o fiscal não precisa. Não lembro exatamente, mas tive algum problema no cadastro de parceiros sem ter ela instalada.

Reply:
Sim precisa sim porque o l10n_br_zip é o que valida o endereço, a NFS precisa de um endereço obrigatório de cobrança, então infelizmente graças ao nosso brasil 7x1 precisamos de Correio pra emitir uma NFS vai enteder. Brasiooooo